README.md

```diff
@@ -44,6 +45,40 @@ Below are all the notebook modules used in our study.
 | [3.jump_analysis](./notebooks/3.jump_analysis/) | Applies our model to the JUMP dataset to predict cellular injuries |
 | [4.visualizations](./notebooks/4.visualizations/) | Contains a notebook responsible for generating our figures |
 
+## Installing respoitory and dependencies
+
+This installation guide assumes that you have Conda installed. If you do not have Conda installed, please follow the documentation [here](https://conda.io/projects/conda/en/latest/user-guide/install/index.html).
+
+1. **Clone the repository**: Clone the repository into your local machine and change the directory to the repo.
+
+    ```bash
+    git clone git@github.com:WayScience/Cytotoxic-Nuisance-Metadata-Analysis.git && cd Cytotoxic-Nuisance-Metadata-Analysis
+    ```
+
+2. **Install dependencies**: There are two environment files in this repository. One is in the root project folder, containing all the necessary packages to conduct our analysis. The other is within the `./notebooks/4.visualization` directory, which includes packages for generating plots with R.
```

Comment:
Maybe consider adding that the analysis is conducted in `Python`, but might just be something the reader should already assume.